File: problem001/answer001.pl
#!/usr/bin/perl

use strict;
use warnings;

my $sum = 0;
for my $n (1..999) {
	if ( ($n % 3) == 0 or ($n % 5) == 0) {
		$sum += $n;
	}
}

print "ANSWER:$sum\n";

File: problem001/answer001.py
#!/usr/bin/python
"""chicks' answer to Euler Project problem #1"""

accumulator = 0
for n in xrange(1, 1000):
	if (n % 3) == 0 or (n % 5) == 0:
		accumulator += n

print "ANSWER:" + str(accumulator)
